finans/kladdeliste.php

```diff
@@ -41,21 +41,23 @@
 include("../includes/online.php");
 include("../includes/topline_settings.php");
 
-if (!isset ($_COOKIE['saldi_kladdeliste'])) $_COOKIE['saldi_kladdeliste'] = NULL;
-
-$sort=isset($_GET['sort'])? $_GET['sort']:Null;
-$rf=isset($_GET['rf'])? $_GET['rf']:Null;
-$vis=isset($_GET['vis'])? $_GET['vis']:Null;
-print "<meta http-equiv=\"refresh\" content=\"150;URL=kladdeliste.php?sort=$sort&rf=$rf&vis=$vis\">";
-
-if (isset($_GET['sort'])) {
+list($sort,$rf,$vis) = [
+	isset($_GET['sort']) ? $_GET['sort'] : NULL,
+	isset($_GET['rf']) ? $_GET['rf']: NULL,
+	isset($_GET['vis']) ? $_GET['vis']: NULL
+];
+if (isset($sort)) {
 	$cookievalue="$sort;$rf;$vis";
 	setcookie("saldi_kladdeliste", $cookievalue, strtotime('+30 days'));
-} else list ($sort,$rf,$vis) = array_pad(explode(";", $_COOKIE['saldi_kladdeliste']), 3, null);
-if (!$sort) {
+} elseif (isset($_COOKIE['saldi_kladdeliste'])) {
+	list($sort,$rf,$vis) = array_pad(explode(";", $_COOKIE['saldi_kladdeliste']), 3, NULL);
+}
+else {
 	$sort = "id";
 	$rf = "desc";
 }
+print "<meta http-equiv=\"refresh\" content=\"150;URL=kladdeliste.php?sort=$sort&rf=$rf&vis=$vis\">";
```

Comment:
I don't know, why you are updating the page every 2½ seconds.
A html meta tag should no go inside a body tag as it does here.

Reply:
I think this had something to do with always keeping the kladdeliste up to date if you were working with other people